src/main/java/com/onarandombox/multiverseinventories/DataStrings.java

```diff
@@ -73,6 +73,10 @@
      * Player bed spawn location identifier.
      */
     public static final String PLAYER_PROFILE_TYPE = "profileType";
+    /**
+     * Player max health identifier.
+     */
+    public static final String PLAYER_MAX_HEALTH = "mhp";
```

Comment:
Perhaps a slightly more description string? `maxHp` or something?

Reply:
Yea possible, I was just following how the other stats named, such as Total experience being txp. But if you prefer maxHp instead, will change it.